src/modules/75-cd/components/Services/ServicesContent/ServicesContent.tsx

```diff
@@ -77,6 +85,22 @@ export const ServicesContent: React.FC = () => {
     lazy: !CDC_DASHBOARD_ENHANCEMENT_NG
   })
 
+  //Service Growth Trend properties
+  const getServicesGrowthTrendQueryParams: GetServicesGrowthTrendQueryParams = useMemo(
```

Comment:
```suggestion
  const servicesGrowthTrendQueryParams: GetServicesGrowthTrendQueryParams = useMemo(
```

As it's not a function, but a const object carrying some values.